packages/adapters/README.md

```diff
@@ -1,4 +1,4 @@
-# Scramjet Transform Hub Adapters
+ # Scramjet Transform Hub Adapters
 
 This module holds the docker adapters utilized by [Scramjet Transform Hub](https://github.com/scramjetorg/transform-hub).
```

Comment:
This module holds two types of adapters utilized by [Scramjet Transform Hub](https://github.com/scramjetorg/transform-hub): Instance Adapter and Sequence Adapter. These Adapters allows for running the Sequence identification and Instance execution in two basic modes: as a non containerized standalone processes or in a Docker container.

Reply:
Added to `readme.mtpl` in adapters package. Thanks Ala! nice catch ⚾